libraries/src/Document/HtmlDocument.php

```diff
@@ -778,6 +787,50 @@ protected function _fetchTemplate($params = [])
         return $this;
     }
 
+    /**
+     * Returns a toolbar object or null
+     *
+     * @param $toolbar
+     * @return Toolbar|null
+     */
+    public function getToolbar($toolbar = 'toolbar', $create = true): ?Toolbar
```

Comment:
```suggestion
     * @param   string   $toolbar
     * @param   boolean  $create
     *
     * @return  ?Toolbar
     *
     * @since   __DEPLOY_VERSION__
     */
    public function getToolbar(string $toolbar = 'toolbar', bool $create = true): ?Toolbar
```
param `$create`
param datatype
since tag

should `$toolbar` forced to be a string?